examples/advanced/vertical_xgboost/jobs/vertical_xgboost_base/app/config/config_fed_client.json

```diff
@@ -0,0 +1,76 @@
+{
+  "format_version": 2,
+  "executors": [
+    {
+      "tasks": [
+        "PSI"
+      ],
+      "executor": {
+        "id": "psi_executor",
+        "name": "PSIExecutor",
+        "args": {
+          "psi_algo_id": "dh_psi"
+        }
+      }
+    },
+    {
+      "tasks": [
+        "train"
+      ],
+      "executor": {
+        "id": "xgb_hist_executor",
+        "name": "FedXGBHistogramExecutor",
+        "args": {
+          "num_rounds": 100,
+          "early_stopping_rounds": 2,
+          "xgb_params": {
+            "validate_parameters": true,
+            "eta": 0.1,
+            "gamma": 1.0,
+            "max_depth": 8,
+            "min_child_weight": 100,
+            "tree_method": "approx",
+            "grow_policy": "depthwise",
+            "eval_metric": "auc"
+          },
+          "data_loader_id": "dataloader"
+        }
+      }
+    }
+  ],
+  "task_result_filters": [],
+  "task_data_filters": [],
+  "components": [
+    {
+      "id": "dataloader",
+      "path": "vertical_data_loader.VerticalDataLoader",
+      "args": {
+        "data_split_path": "/tmp/nvflare/vertical_xgboost/data/site-x/higgs.data.csv",
+        "label_owner": "site-x",
+        "train_proportion": 0.8
+      }
+    },
+    {
+      "id": "dh_psi",
+      "name": "DhPSITaskHandler",
+      "args": {
+        "local_psi_id": "local_psi"
+      }
+    },
+    {
+      "id": "local_psi",
+      "path": "local_psi.LocalPSI",
+      "args": {
+        "psi_writer_id": "psi_writer",
+        "data_split_path": "/tmp/nvflare/vertical_xgboost/data/site-x/higgs.data.csv"
+      }
+    },
+    {
+      "id": "psi_writer",
+      "name": "FilePSIWriter",
+      "args": {
+        "output_path": "psi/intersection.txt"
```

Comment:
this would write out to "[app_dir]/psi/intersection.txt"

Reply:
In the FilePSIWriter the output path is as follows:
`job_dir = os.path.dirname(os.path.abspath(fl_ctx.get_prop(FLContextKey.APP_ROOT)))`
`return os.path.join(job_dir, fl_ctx.get_identity_name(), self.output_path)`
(https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_common/psi/file_psi_writer.py#L64)

Will keep this for now and possibly discuss changing in the future, as the FilePSIWriter is also used in other places.